### ISSUE TITLE
Allow ERC20 transfers through the Optimism Gateway

### ISSUE THREAD

**Issue by ubaidurk** [ethereum-optimism/community-hub#129]
Awesome work on launch today.

The optimism gateway (https://gateway.optimism.io/select-asset) doesn't allow for transferring any arbitrary ERC20 token through the bridge. I'm confused as to why.

Would be awesome if you could make this change.

**Comment by elenadimitrova** [comment]
This is definitely on the roadmap! We'll update the community documentation here with better information on gateway plans.

**Comment by mr-stegosaurus** [comment]
Is it still possible to pass an asset through with command line tools?

**Comment by elenadimitrova** [comment]
@mr-stegosaurus yes you can call the bridge interface methods, defined here https://github.com/ethereum-optimism/optimism/tree/master/packages/contracts/contracts/optimistic-ethereum/iOVM/bridge/tokens 
Overview of the standard bridge here https://community.optimism.io/docs/developers/bridge/standard-bridge.html#depositing-eth

L1 and L2 bridge addresses are liked here https://community.optimism.io/docs/infra/networks.html
